docs/canvas/canvas-workpad.asciidoc

```diff
@@ -1,72 +1,93 @@
 [role="xpack"]
-[[canvas-workpad]]
-=== Create a workpad
+[[create-canvas-workpad]]
+== Create a workpad
 
-When you create a workpad, you'll start with a blank page, or you can choose a workpad that is already started for you. 
+A Canvas _workpad_ provides you with a workspace where you can build presentations of your live data.
 
-. Open *Canvas*.
+To create a workpad, you can:
 
-. Choose one of the following options:
+* <<blank-canvas-workpad,Build fully customizable presentations of your own data>>
 
-* To start with a single, blank workpad, click *Create workpad*.
+* <<canvas-template-workpad,Choose from a set of preconfigured workpad templates>>
 
-* To create a workpad from a template, select *Templates*, then select a preconfigured template.
+* <<import-canvas-workpad,Import a workpad that someone else has already started>>
 
-* To import an existing workpad, click and drag a workpad JSON file to the *Import workpad JSON file* field.
+* <<sample-data-workpad,Gather inspiration from the {kib} sample data workpads>>
 
-For advanced workpad examples, add a {kibana-ref}/add-sample-data.html[sample Kibana data set], then select *Canvas* from the *View Data* dropdown list.
+[float]
+[[blank-canvas-workpad]]
+=== Start with a blank page
 
-For more workpad inspiration, go to the link:https://www.elastic.co/blog/[Elastic Blog]. 
+To use the background colors, images, and data of your choice, start with a blank workpad. 
 
-[float]
-=== Personalize your workpad
+. Open *Canvas*.
 
-To personalize your workpad, use the editor on the right.
+. On the *Canvas workpads* page, click *Create workpad*.
 
-. Add a *Name* to your workpad, for example, `My Canvas Workpad`.
+. Add a *Name* to your workpad.
 
-. In the *Width* and *Height* fields, specify the size of your workpad. 
+. In the *Width* and *Height* fields, specify the size. 
 
-. Select the layout for your workpad. 
+. Select the layout. 
```

Comment:
We should add this to the end or the previous set. The user can either enter a custom width/height or click on the preset dimensions, but they shouldn't need to do both.